Traceback (most recent call last):
  File "pred_train.py", line 119, in <module>
  File "pred_train.py", line 112, in main
    train_losses.append(train(model, train_loader))
  File "pred_train.py", line 50, in train
    for measurements,images in dataloader:
  File "/kuacc/users/ajamshidi18/.conda/envs/driving2/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 435, in __next__
    data = self._next_data()
  File "/kuacc/users/ajamshidi18/.conda/envs/driving2/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 475, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/kuacc/users/ajamshidi18/.conda/envs/driving2/lib/python3.7/site-packages/torch/utils/data/_utils/fetch.py", line 44, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/kuacc/users/ajamshidi18/.conda/envs/driving2/lib/python3.7/site-packages/torch/utils/data/_utils/fetch.py", line 44, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/scratch/users/ajamshidi18/hpc_run/AutoDriving/hw1/expert_dataset.py", line 25, in __getitem__
    rgb = image.convert("RGB")
  File "/kuacc/users/ajamshidi18/.conda/envs/driving2/lib/python3.7/site-packages/PIL/Image.py", line 933, in convert
    self.load()
  File "/kuacc/users/ajamshidi18/.conda/envs/driving2/lib/python3.7/site-packages/PIL/ImageFile.py", line 269, in load
    n, err_code = decoder.decode(b)
KeyboardInterrupt